### DISCUSSION TITLE
import from synced storage in firefox for android

### DISCUSSION THREAD

**Discussion by nexdep** [proginosko/LeechBlockNG#443] [Q&A] [answered]
I am trying to automatically share the same settings across my devices and I noticed that the import from synced storage option is not present in Android. Is it possible to implement it someday? It would also be useful to have time limits that are enforced cross platform

**Comment by proginosko** [comment, marked as answer]
Unfortunately, Android does not (yet) properly support sync storage.

**Reply by nexdep** [reply]
ok, thanks a lot!

**Reply by nexdep** [reply]
I made a (tiny) donation - I am starting to find this helpful
